src/dbFirebase.js

```diff
@@ -141,6 +142,7 @@ async function writeFeedback(data) {
   if (data.latitude && data.longitude) {
     data.location = new firebase.firestore.GeoPoint(data.latitude, data.longitude);
   }
+  data.device = device();
```

Comment:
can we pass the device as parameter ? in this way, dbFirebase will not care about it (single responsibility)